src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -79,9 +79,9 @@
   (let [blur-amount   (header-animations/use-blur-amount scroll-amount)
         layer-opacity (header-animations/use-layer-opacity
                        scroll-amount
-                       "transparent"
+                       (colors/theme-colors colors/white-opa-0 colors/neutral-95-opa-0 theme)
```

Comment:
When we animate from `transparent` to white, the animation goes through black, so now the starting color is the ending color with opacity zero.

cc: @ajayesivan